.travis.yml

```diff
@@ -45,16 +45,21 @@ script:
 
 jobs:
   include:
-    # Define the commitlint stage
-    - stage: commitlint
+    # Define the `lint` stage (runs `yamllint` and `commitlint`)
+    - stage: lint
       language: node_js
       node_js: lts/*
       before_install: skip
       script:
+        # Install and run `yamllint`
+        - pip install --user yamllint
+        # yamllint disable-line rule:line-length
+        - yamllint -s . .yamllint pillar.example test/salt/pillar/default.sls test/salt/pillar/fedora.sls
```

Comment:
will this line need to be maintained separately for each formula, depending of pillar files provided?

Reply:
@daks "Great minds think alike" -- let me refer you to the query and explanation that took place from here: https://github.com/saltstack-formulas/apt-formula/pull/49#discussion_r311670351.

Reply:
ok, nice :)